From 50920dea2f2ff29459142230a05bb3b7d76afd2c Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 19:49:56 +0800
Subject: [PATCH 0119/5242] MLK-9637-2 usb: otg: test: add otg_srp_reqd flag
 in otg_fsm

commit  2712897d0bc50a72658e6f081fc6d798dea8a069 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds flag otg_srp_reqd in struct otg_fsm to indicate A device
as host wants B-UUT to do SRP once it ends the session.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 8e5f1b6b12ec77ec591909b09727049ffff37b9c)
(cherry picked from commit 7eaf8cf8ff375917e9972cca69bcecd6d807e6db)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/usb/otg-fsm.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 0c8b3cc..1655fdc 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -177,6 +177,7 @@ struct otg_fsm {
 	/* OTG test device */
 	int tst_maint;
 	int otg_vbus_off;
+	int otg_srp_reqd;
 
 	/* Informative variables. All unused as of now */
 	int a_bus_drop_inf;
-- 
1.7.9.5

